### DISCUSSION TITLE
Why is there no javaw.exe in jdk and jre for windows?

### DISCUSSION THREAD

**Discussion by kdaek21** [ikvmnet/ikvm#614] [Q&A]
In the case of the Windows version of the general JDK or JRE, javaw.exe is provided so that bytecode can be executed without the console if the console is not desired. However, in the case of IKVM, only java.exe is provided for dynamic conversion.

Why is javaw.exe not provided?
(Is it because javaw.exe is not provided separately because you can use ikvmc.exe to statically convert to winexe if you do not want the console?)

**Comment by wasabii** [comment]
It's not provided simply because I never added it. AWT has only existed for a single release.

**Comment by kdaek21** [comment]
I made a simple AWT program and ran it, and it ran well, so I confirmed that AWT is supported. (I haven't tested it, but if it implements everything in Java8, it seems that Swing will be supported. I don't know about JavaFX, though.)

If you don't plan to provide javaw.exe for the Windows version, I will close this question by converting it statically to winexe for cases where the console is not needed.

**Reply by wasabii** [reply]
I don't know if I plan to support or. Should I?

You know, this isn't a hard one to add for an enterprising developer.......
